Libraries/plist/Tools/PlistBuddy.cpp

```diff
@@ -586,5 +602,5 @@ main(int argc, char **argv)
         }
     }
 
-    return 0;
+    return (success ? 0 : 1);
```

Comment:
What's the point of this? `success` is a `bool`, and this function returns a `bool`.